### ISSUE TITLE
Multiple Column Module doesn't work

### ISSUE THREAD

**Issue by elfeffe** [microweber/microweber#516]
Please check the screenshot
https://gyazo.com/d5aa84ce52f50ef4e2c34a7108cea9d4

It must add the "mw-empty element" with an ID, inside the "mw-col-container element"
But if you see the second column, it added the ID to the mw-col div, and there is no "mw-empty element".
So drag and drop doesn't work until you fix the HTML using Chrome Dev Tools.
The columns look like https://gyazo.com/996e1bf8ea7755573da41cc8aad7386d

I fixed HTML and now it looks liks this
https://gyazo.com/845b77b38d4aeec97504b8d18770f955

element ID is element_1548409152392, check in both screenshots
The columns look like https://gyazo.com/21e64f40909c6da20151a05aaa7bdb50

It looks like the javascript, the Drag&Drop system, doesn't work very good. It creates arbitrary elements everywhere.

**Comment by peter-mw** [comment]
fixed at commit b6b650949740a08a3602cfc2209d570786ca981d

**Comment by elfeffe** [comment]
I want to trest it, but my tools.js file doesn't update.
I have downloaded DEV branch, and pasted over the original microweber files.
I don't know how to force tools.js reload because it is loaded using mw.require('tools.js', true);
I'm getting
Message: TypeError: mw.tools.distance is not a function. (In 'mw.tools.distance(mw.handlerMouse.x, mw.handlerMouse.y, mw.emouse.x, mw.emouse.y)', 'mw.tools.distance' is undefined)

**Comment by elfeffe** [comment]
ok, deleter /userfiles/cache content and it works

**Comment by elfeffe** [comment]
@peter-mw will you merge it someday?
Your latest release is from 2016, and master branch is full of bugs. Do you plan to merge dev with master and release a new version?

**Comment by peter-mw** [comment]
yes will merge it soon and we will release new version in few days
